webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+
+const isDev = process.env.NODE_ENV === 'development';
+
+module.exports = {
+    mode: isDev ? 'development' : 'production',
+    entry: './src/index.ts',
+    output: {
+        path: path.join(__dirname, 'umd'),
+        library: {
+            name: 'PrezlyUploadcare',
+            type: 'umd',
```

Comment:
That's unexpected ([POLA](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) violation).
The standard is to put **all** built things into a single directory (usually: `build`). And then create subdirectories in it. 

See how react does it:
![image](https://user-images.githubusercontent.com/6830683/116074811-6dbb8380-a6bc-11eb-97ca-c33ddd50bff8.png)

It's possible thanks to `files` attribute in `package.json`: https://docs.npmjs.com/cli/v6/configuring-npm/package-json#files

Reply:
I was going with the vague guidelines the https://unpkg.com/ provided (umd build was created only for hosting on unpkg). I guess it should also work with `build/umd` folder, but somehow I didn't check that possibility :(

I don't think there will be anymore changes to this package in a near future, so I created a task to tackle both of your comments later on:
https://linear.app/prezly/issue/DEV-2680/prezlyuploadcare-address-feedback-on-umd-build